_posts/2018-08-20-how-we-use-pivotal-tracker-at-ombu-labs.markdown

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "How we use Pivotal Tracker at Ombu Labs"
+date: 2018-08-20 11:10:00
+categories: ["agile", "pivotal-tracker"]
+author: "etagwerker"
+---
+
+We tend to use [GitHub](https://github.com) for everything at [Ombu Labs](https://www.ombulabs.com), but sometimes our clients
+prefer [Pivotal Tracker](https://www.pivotaltracker.com). We have no problem
+with Pivotal Tracker, we just like to keep everything in one place, that is why
+we prefer GitHub for everything.
+
+There are many things that are easier with Pivotal Tracker, as long as you are
+using it the right way. Some of its features are very useful for agile teams like
+ours. This is how we like to use it to ship value with every sprint and keep
+track of our [velocity](http://wiki.c2.com/?ProjectVelocity).
+
+<!--more-->
+
+## Step 1: Filing user stories
+
+Every thing has to be tracked according to the type of story. Most of the user
```

Comment:
It's weird seeing `Every thing has to be` separate, I think `Everything has to be` is more natural.

Reply:
It'd make sense if it said `Every single thing`.